### PR TITLE
fixed bug that caused state revert in darkmode button

### LINKED ISSUE #351: Dark mode button resets to its default state instead of maintaining the set theme
### Describe the bug

Upon clicking the browser's back button, the theme toggler icon reverts to its initial state, rather than adjusting based on the theme as intended.

### Expected result

The icon ought to be configured in accordance with the chosen theme.

### Reproduction

After enabling dark mode on the website, navigating back to the home page of the code generator and then revisiting the "Get started" page results in the icon being altered, contrary to its intended behavior.

### Steps to reproduce

1. Navigate to the code generator page and click "Get Started."
1. Switch the theme to dark mode and use the back button to return to the main webpage.
1. Upon re-entering the "Get Started" page, the icon will reset, functioning contrary to the intended behavior.

### PR DESCRIPTION
**Pull request by avinashsharma080**
### Description

Fix #351

Added an if-else statement that checks the theme of the website and sets the button state according to that theme.

### What is the purpose of this pull request?
-  Bug fix

### CODE REVIEW

**Comment by avinashsharma080** [comment]
@vfdev-5 please check this new pr

**Comment by vfdev-5** [comment]
I have few questions about the dark mode implementation that I haven't spotted previously:
1) `data()` returns a dict with key: `userTheme`. Are we using it somewhere or we can remove it?
2) In the code you are doing `activeTheme === 'light-theme'` and in the PR you do `localStorage.getItem('user-theme') == 'light-theme'`, which one is correct and why ?
3) the line `isDark: true ? localStorage.getItem('user-theme') == 'light-theme' : false` is very confusing in the following sense: in case local storage has user theme equals "light-theme" why we set `isDark=true`. Can we simplify the namings : two notions: user-theme and isDark into a single notion for example `isDark`?

Can you close the other one if this one superseeds it?

**Comment by avinashsharma080** [comment]
1.We can remove userTheme, it was being used earlier , now we don't need it.
2. I created a variable for comparison but we can directly check if localStorage.getItem('user-theme') is 'light-theme', should I remove the variable ? Both are correct
3. I renamed the variable to isLight to simplify naming 

I can raise a PR with the following changes

**Comment by vfdev-5** [comment]
About 2), my point about using `==` vs `===`. Anyway, let's remove all unused vars and simplify the code such that it reads easily

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "@pytorch-ignite/code-generator",
   "version": "0.4.0",
   "scripts": {
-    "dev": "vite --port 5000",
+    "dev": "vite --port 5001",
```
Comment (vfdev-5) [inline package.json]: Let's revert this change

Comment (avinashsharma080) [reply package.json]: Yep I did this in next commit

**Review by vfdev-5 — APPROVED** [review]
Thanks for the fix @avinashsharma080 !
